### PR TITLE
Add "Your Profile" link to user dropdown

### PR DESCRIPTION
**Pull request by berkerpeksag**
Currently, the only way I found to go to your public
profile is to click the "public profile" links at
https://pypi.org/manage/account/

### CODE REVIEW

**Comment by nlhkabu** [comment]
Hi @berkerpeksag thanks for this PR :)

What is the benefit of linking prominently to this page from here?  All of the information on this page is available via the management UI (with more detail).  Is there something missing from the management UI that you get on this page?

I also think we need to test this with some users before merging it - my concern is that the user might confuse the 'profile' link with the 'account settings' link.

**Comment by berkerpeksag** [comment]
The management UI is only for me so I can't share it on my Patreon page or add it to my CV. Also, adding a link to a user's public profile is a common feature on many websites:

python.org (see "View your public profile"):



GitHub (see "Your Profile"):



Twitter (see "Profile"):



Last.fm (see "View profile"):

File: warehouse/templates/includes/current-user-indicator.html [hunk 1]
```diff
@@ -29,6 +29,10 @@
           Admin
         </a>
         {% endif %}
+        <a class="dropdown__link" href="{{ request.route_path('accounts.profile', username=request.user.username) }}">
+          <i class="fa fa-cube" aria-hidden="true"></i>
```
Comment (di) [inline warehouse/templates/includes/current-user-indicator.html]: This should probably have a different icon than the "Your Projects" link.

Comment (berkerpeksag) [reply warehouse/templates/includes/current-user-indicator.html]: Good point, thanks. Do you have any suggestions?

Comment (di) [reply warehouse/templates/includes/current-user-indicator.html]: We use [`user-circle`](https://fontawesome.com/icons/user-circle?style=solid) for this elsewhere, that might be the best choice but I'll leave it to @nlhkabu.

File: warehouse/templates/includes/current-user-indicator.html [hunk 2]
```diff
@@ -29,6 +29,10 @@
...
+          Your Profile
```
Comment (berkerpeksag) [inline warehouse/templates/includes/current-user-indicator.html]: Should I rename this to "Your Public Profile"?

File: warehouse/templates/includes/current-user-indicator.html (hunk 1 above)
Comment (nlhkabu) [reply warehouse/templates/includes/current-user-indicator.html]: +1 for `user-circle`

File: warehouse/templates/includes/current-user-indicator.html (hunk 2 above)
Comment (nlhkabu) [reply warehouse/templates/includes/current-user-indicator.html]: If it fits, that would be better @berkerpeksag

Comment (nlhkabu) [reply warehouse/templates/includes/current-user-indicator.html]: @berkerpeksag if "Your public profile" does not fit, I think "Public profile" would suffice.  IMO, it's clearer than "Your profile"

**Comment by berkerpeksag** [comment]
Superseded by #3255. I accidentally deleted the fork in my profile instead of an organization so I couldn't edit this PR via GitHub.

**Pull request closed without merging**